cfg/excalibur/jec17ul/data17_09Aug2019/common.py

```diff
@@ -1,7 +1,7 @@
 JEC_BASE='Summer19UL17'
 JEC_VERSION='V5'
 
-JER = 'Summer19UL17_JRV2' # set this to 'None' to turn JER smearing off
+JER = None
```

Comment:
Why did you turn off the JER smearing for UL17?

Reply:
This is the data config. JER smearing should only run for MC.

Reply:
Sorry, didn't see that it is the data config. Seems to be fine then.